sample/src/main/java/com/google/accompanist/sample/navigation/material/BottomSheetNavSample.kt

```diff
@@ -101,11 +101,15 @@ private fun HomeScreen(showSheet: () -> Unit, showFeed: () -> Unit) {
 
 @Composable
 private fun BottomSheet(showFeed: () -> Unit, showAnotherSheet: () -> Unit, arg: String) {
-    Text("Sheet with arg: $arg")
-    Button(onClick = showFeed) {
-        Text("Click me to navigate!")
-    }
-    Button(onClick = showAnotherSheet) {
-        Text("Click me to show another sheet!")
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Text("Sheet with arg: $arg")
+        Button(onClick = showFeed) {
+            Text("Click me to navigate!")
+        }
+        Button(onClick = showAnotherSheet) {
+            Text("Click me to show another sheet!")
+        }
```

Comment:
@jossiwolf Using a `column` composable wraps the elements and lays them out in a column. Thus avoiding the issue.

![image](https://user-images.githubusercontent.com/63651093/142645183-6e48bf38-1aa5-4b94-b5a7-201fdb51fa78.png)